website/pages/en/developing/creating-a-subgraph.mdx

```diff
@@ -527,13 +527,37 @@ The first handler takes a `NewGravatar` event and creates a new `Gravatar` entit
 
 The second handler tries to load the existing `Gravatar` from the Graph Node store. If it does not exist yet, it is created on-demand. The entity is then updated to match the new event parameters before it is saved back to the store using `gravatar.save()`.
 
-### Recommended IDs for Creating New Entities
+### Using Bytes as IDs
 
-Every entity has to have an `id` that is unique among all entities of the same type. An entity's `id` value is set when the entity is created. Below are some recommended `id` values to consider when creating new entities. NOTE: The value of `id` must be a `string`.
+You can use `Bytes` as the type for the `id` field of entities, and it is highly recommended to use `Bytes` wherever possible, and only use `String` for attributes that truly contain human-readable text, like the name of a token.
```

Comment:
I think the alternative to `Bytes` is `ID`, which translates to `String`. Can you validate?

Reply:
The ID often corresponds to a string value. However, depending on the data source and implementation, it could also be another data type.